specification/storage/resource-manager/Microsoft.Storage/preview/2018-03-01-preview/examples/StorageAccountGetManagementPolicy.json

```diff
@@ -1,75 +1,57 @@
 {
   "parameters": {
     "subscriptionId": "45b60d85-fd72-427a-a708-f994d26e593e",
-    "resourceGroupName": "res7231",
-    "accountName": "sto288",
+    "resourceGroupName": "res6977",
+    "accountName": "sto2527",
     "managementPolicyName": "default",
     "api-version": "2018-03-01-preview",
     "monitor": "true"
   },
   "responses": {
     "200": {
       "body": {
-        "id": "/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/resourceGroups/res7231/providers/Microsoft.Storage/storageAccounts/sto288/managementPolicies/default",
+        "id": "/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/resourceGroups/res6977/providers/Microsoft.Storage/storageAccounts/sto2527/managementPolicies/default",
         "name": "DefaultManagementPolicy",
         "type": "Microsoft.Storage/storageAccounts/managementPolicies",
         "properties": {
           "policy": {
-            "version": 0.5,
+            "version": "0.5",
             "rules": [
               {
-                "name": "ruleFoo",
-                "type": "lifecycle",
+                "name": "olcmtest",
+                "type": "Lifecycle",
                 "definition": {
                   "filters": {
                     "blobTypes": [
                       "blockBlob"
                     ],
-                    "nameMatch": [
-                      "foo"
+                    "prefixMatch": [
+                      "olcmtestcontainer"
                     ]
                   },
                   "actions": {
                     "baseBlob": {
                       "tierToCool": {
-                        "daysAfterLastModifiedGreaterThan": 30
+                        "daysAfterModificationGreaterThan": 1000
```

Comment:
hi Wei, small issue here. TierToCool days should be smaller than TierToArchive days, otherwise there would be a run time error. can you make change it from 1000 to 30?